.github/workflows/rosa-scaling-benchmark.yml

```diff
@@ -193,6 +193,9 @@ jobs:
           ROSA_CLUSTER_NAME: ${{ inputs.clusterName || format('gh-{0}', github.repository_owner) }}
           NAMESPACE: ${{ env.PROJECT }}
 
+      - name: Restart Keycloak Pods after dataset creation
+        run: kubectl delete pods -n ${{ env.PROJECT }} -l app=keycloak
```

Comment:
I assume this works, as the creation of the EC2 instance takes longer than restarting the Keycloak instances. Still, for the sake of consistency, I'd like to wait using `kubectl wait` like we do it in the isup.sh script, or even use the isup.sh script itself if it would apply here.